### PR TITLE
Add olg.ca

### PR DESCRIPTION
**Pull request by thomassth**

### CODE REVIEW

**Comment by hkamran80** [comment]
Here's a vector image you can use:
OLG

**Review by hkamran80 — CHANGES_REQUESTED** [review]
You can use a vector image:
OLG

File: img/o/olg.ca.svg [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            <path fill="#003C71" d="M.1 9.56h5.33V.07H.1z" mask="url(#l)" />
+        </g>
+        <g transform="translate(36.4 18.97)">
+            <mask id="n" fill="#fff">
+                <use xlink:href="#m" />
+            </mask>
+            <path fill="#003C71" d="M.07 9.83h7.85V0H.07z" mask="url(#n)" />
+        </g>
+    </g>
+</svg>
```
Comment (hkamran80) [inline img/o/olg.ca.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 50 40"><defs><path id="b" d="M8 15.9A7.95 7.95 0 0 1 .01 7.96 7.95 7.95 0 0 1 8 .02c4.4 0 7.97 3.56 7.97 7.94A7.95 7.95 0 0 1 8 15.9Z"/><path id="e" fill="#FAE300" d="M.01 15.91h15.96V.02H.01z" mask="url(#a)"/></defs><defs><path id="c" d="M4.78 39.99H0L.06.02l39.79 39.92z"/><path id="j" d="M4.47.08C6.91.08 8.9 2.3 8.9 5.02S6.91 9.95 4.47 9.95C2.03 9.95.05 7.75.05 5.02.05 2.29 2.03.08 4.47.08Zm-.02 8.4c1.46 0 2.65-1.54 2.65-3.43 0-1.89-1.19-3.43-2.65-3.43S1.8 3.16 1.8 5.05c0 1.9 1.19 3.43 2.65 3.43Z"/><path id="l" d="m.12.07 1.86.05.04 7.77 3.11-.04.3 1.7-5.32-.01z"/><path id="n" d="m4.18 4.3.03.02 3.7-.02v4.53c-2.55 1.54-4.63.82-4.63.82a4 4 0 0 1-3-3.13c-.6-2.53.37-4.21.37-4.21C1.59.45 3.19.19 3.19.19c2.2-.64 4.26.54 4.26.54l-.27 1.7C4.58.69 3 2.07 3 2.07c-1.26.91-1.1 3.06-1.1 3.06.1 1.88 1.06 2.6 1.06 2.6 1.44 1.04 3.36.18 3.36.18L6.3 5.8H4.43l-.25-1.5Z"/><use xlink:href="#b" id="f"/><use xlink:href="#b" id="h"/></defs><g fill="none" fill-rule="evenodd"><g transform="translate(9.1)"><mask id="d" fill="#fff"><use xlink:href="#c"/></mask><path fill="#E4002B" d="M0 39.99h39.85V.01H0z" mask="url(#d)"/></g><g transform="translate(.16 16)"><mask id="a" fill="#fff"><use xlink:href="#b"/></mask><use xlink:href="#e" mask="url(#a)"/></g><g transform="translate(16.45 16)"><mask id="g" fill="#fff"><use xlink:href="#f"/></mask><use xlink:href="#e" mask="url(#g)"/></g><g transform="translate(32.73 16)"><mask id="i" fill="#fff"><use xlink:href="#h"/></mask><use xlink:href="#e" mask="url(#i)"/></g><g transform="translate(3.6 18.97)"><mask id="k" fill="#fff"><use xlink:href="#j"/></mask><path fill="#003C71" d="M.05 9.95H8.9V.08H.05z" mask="url(#k)"/></g><g transform="translate(21.95 19.2)"><mask id="m" fill="#fff"><use xlink:href="#l"/></mask><path fill="#003C71" d="M.1 9.56h5.33V.07H.1z" mask="url(#m)"/></g><g transform="translate(36.4 18.97)"><mask id="o" fill="#fff"><use xlink:href="#n"/></mask><path fill="#003C71" d="M.07 9.83h7.85V0H.07z" mask="url(#o)"/></g></g></svg>
```

Here's a minified SVG

**Comment by thomassth** [comment]
Thanks!

Where did you find the SVG by the way?

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by hkamran80** [comment]
> Thanks!
> 
> Where did you find the SVG by the way?

No problem! I went to their website, looked at the images with their logos using Inspect Element, and found one with an SVG file extension. I then minified it with [SVGOMG](https://jakearchibald.github.io/svgomg/).

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**